pom.xml

```diff
@@ -32,7 +32,13 @@
   </properties>
 
   <dependencies>
-
+	
+	<dependency>
```

Comment:
outdent 2 spaces, so that it's consistent with the end tag (and other dependency tags)